client/src/components/utils/Footer/style.css

```diff
@@ -0,0 +1,96 @@
+.footer {
+  background: var(--main-color--light);
+  color: var(--main-white);
+  display: flex;
+  flex-direction: column;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.footer h2 {
+  color: var(--main-white);
+  font-weight: 700;
+  font-size: 1.75rem;
```

Comment:
since you are modifying the header's `font-weight` and `font-size` then there is no need to use `h2` tag